lib/ontologies_linked_data/models/mod/semantic_artefact_distribution.rb

```diff
@@ -2,62 +2,64 @@ module LinkedData
     module Models
 
         class SemanticArtefactDistribution < LinkedData::Models::Base
-            
-            class << self
-                attr_accessor :attribute_mappings
-                def attribute_mapped(name, **options)
-                  mapped_to = options.delete(:mapped_to)
-                  attribute(name, **options)
-                  @attribute_mappings ||= {}
-                  @attribute_mappings[name] = mapped_to if mapped_to
-                end
-            end
+            include LinkedData::Concerns::AttributeMapping
```

Comment:
Already a lot better,
just here put your concerns in module `SemanticArtefact`, so that here it will be called like this `LinkedData::Concerns::SemanticArtefact::AttributeMapping` (and same for the fetcher), 
otherwise we will think that we can use it for everything.

Reply:
done